airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1354,6 +1354,68 @@ def run_grant_dataset_view_access(self,
                 view_project, view_dataset, view_table, source_project, source_dataset)
             return source_dataset_resource
 
+    def create_empty_dataset(self, dataset_id="", project_id="", dataset_resource=None):
+        """
+        Create a new empty dataset:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/insert
+
+        :param project_id: The name of the project where we want to create
+               an empty a dataset. Not need to provide, if projectId in dataset_resource.
```

Comment:
Just need one tab in docstring, see following for example:

https://github.com/apache/incubator-airflow/blob/b04158792e82ab799c6cd648ffd2d34dbf069e1e/airflow/contrib/operators/bigquery_operator.py#L32-L37

```
        :param project_id: The name of the project where we want to create
           an empty a dataset. Not need to provide, if projectId in dataset_resource.
```